### PR TITLE
M3-5580: Update Storybook Entity Table

### PR DESCRIPTION
**Pull request by HanaXu**
## Description
Convert file to mdx and add other instances of where EntityTable is used (Volumes/Images/Firewalls/NodeBalancers landing)

## How to test
`yarn storybook` and check the stories under `Components/Entity Table`

### CODE REVIEW

**Review by DevDW — COMMENTED** [review]
> The Volume and NodeBalancer headers are imported from their component but they display as cut off, I'm not sure why

I looked into this and will revisit it later today, but I couldn't quite pinpoint why this is happening. The only way I was able to fix it so far was by creating a new variable with the headers plus the visually hidden header for the Actions column, like

```
  const headers = [
      {
    label: 'Label',
    dataColumn: 'label',
    sortable: true,
    widthPercent: 40,
  },
  {
    label: 'Region',
    dataColumn: 'region',
    sortable: true,
    widthPercent: 15,
  },
  {
    label: 'Size',
    dataColumn: 'size',
    sortable: true,
    widthPercent: 5,
  },
  {
    label: 'Attached To',
    dataColumn: 'Attached To',
    sortable: false,
    widthPercent: 25,
  },
      {
      label: 'Action Menu',
      visuallyHidden: true,
      dataColumn: '',
      sortable: false,
      widthPercent: 5,
    },
  ]
```

and passing that to the `headers` prop.

File: packages/manager/src/components/EntityTable/EntityTable.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+      }}
+      initialOrder={{ order: 'asc', orderBy: 'label' }}
+    />
+  )
+}
+
+# Entity Detail
+<Canvas>
+  <Story name="Volumes">
+    <h2 style={{marginTop: 0}}>Volumes: </h2>
```
Comment (bnussman) [inline packages/manager/src/components/EntityTable/EntityTable.stories.mdx:159]: Can we remove these `<h2 />`s and use Markdown outside of the `<Canvas />` instead?

File: packages/manager/src/components/EntityTable/EntityTable.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+      entity="nodebalancer"
+      headers={nodeBalancerHeaders}
+      row={{
+        Component: NodeBalancerTableRow,
+        data: nodeBalancers,
+        handlers: {},
+        lastUpdated: 1637782175231,
+      }}
+      initialOrder={{ order: 'asc', orderBy: 'label' }}
+    />
```
Comment (bnussman) [inline packages/manager/src/components/EntityTable/EntityTable.stories.mdx:152]: This should fix the visual error with the NodeBalancers table.

```suggestion
    <EntityTable
      entity="nodebalancer"
      toggleGroupByTag={() => false}
      isGroupedByTag={false}
      headers={nodeBalancerHeaders}
      row={{
        Component: NodeBalancerTableRow,
        data: nodeBalancers,
        handlers: {},
        lastUpdated: 1637782175231,
      }}
      initialOrder={{ order: 'asc', orderBy: 'label' }}
    />
```

**Review by bnussman — CHANGES_REQUESTED** [review]

**Review by DevDW — APPROVED** [review]
LGTM 🚢

**Review by bnussman — APPROVED** [review]

**Review by tiffwong — APPROVED** [review]
Great job!

**Pull request merged**